Client/Client.Core/Client.Core.Static/Client.hpp

```diff
@@ -4,6 +4,8 @@
 #include "Network/Message.hpp"
 #include "Network/SafeQueue.hpp"
 
+#include <FileLogger.hpp>
```

Comment:
It's better to move FileLogger header to cpp part. It should improve build time and avoid unnecessary dependencies.